pmm-app/tests/QAN/steps/qanActions.js

```diff
@@ -425,4 +426,16 @@ module.exports = {
 
     return resultsCount[2];
   },
+
+  async verifySelectedFilters(filters) {
+    I.click(qanPage.fields.showSelected);
+    const getFilter = await I.grabTextFrom('span.checkbox-container__label-text');
```

Comment:
probably would be better to have this locator move to fields, and also wait for element before you try to fetch text from it please.